### PR TITLE
Initial commit and addition of stack management bash scripts

### PR DESCRIPTION
**Pull request by amohammedalfresco**

### CODE REVIEW

**Comment by SubatomicHero** [comment]
Is this to build the trial stack rather than build the control system?

File: cloudformation/updateStack-ot-stack.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```
Comment (SubatomicHero) [inline cloudformation/updateStack-ot-stack.sh]: This entire script is pointless actually, we are not planning on running updates on Trial environments

File: cloudformation/ot-stack-parameters.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "ParameterKey":"AdminUsername",
+    "ParameterValue":"admin"
+  },
+  {
+    "ParameterKey":"AdminPassword",
+    "ParameterValue":"alfrescodevops"
+  },
+  {
+    "ParameterKey":"TrialAMIId",
+    "ParameterValue":"ami-b57500a3"
```
Comment (SubatomicHero) [inline cloudformation/ot-stack-parameters.json]: All the values should be replaced using`sed`. The AMI should come from the packer log. The other values should come from bamboo as parameters

File: cloudformation/ot-stack-parameters-update.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
```
Comment (SubatomicHero) [inline cloudformation/ot-stack-parameters-update.json]: This file isnt needed, we are not doing rolling updates/upgrades to the trial stacks

File: cloudformation/online-trial-stack.yaml [hunk 4]
```diff
@@ -354,8 +354,6 @@
                   fi
                   # These commands should probably be Cheffed pas part of AMI creation/config. They currently add 5-7 minutes to the wait time
                   if [[ "$RESULT" == 0 ]]; then
-                    systemctl set-property snortd CPUShares=10 MemoryLimit=256M
-                    /usr/local/bin/reload-snort.sh
                     curl -L -u ${AdminUsername}:${AdminPassword} 'http://${AdminUsername}:${AdminPassword}@localhost:8070/alfresco/service/slingshot/node/search?q=%40name%3A%22Project%20Meeting%20Minutes%22&lang=lucene&store=workspace%3A%2F%2FSpacesStore'
```
Comment (SubatomicHero) [inline cloudformation/online-trial-stack.yaml]: Doesnt the removal of Snort mean we need a new AMI ID in the template?

Comment (amohammedalfresco) [inline cloudformation/online-trial-stack.yaml]: Hey Martin! I'm not done yet.  Will ping you once all done.

**Pull request merged**